### PR TITLE
Add actuator

### PR DESCRIPTION
**Pull request by adrianchiris**

### CODE REVIEW

**Comment by adrianchiris** [comment]
Note: last 3 commits are relevant

File: pkg/tc/actuator_file_writer.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+	klog.Infof("saving new rules to: %s", a.path)
+
+	file, err := os.Create(a.path)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+
+	_, err = newBuf.WriteTo(file)
```
Comment (rollandf) [inline pkg/tc/actuator_file_writer.go:77]: Will this overwrite the old content?
According to [os.Create](https://pkg.go.dev/os#Create) :  `If the file already exists, it is truncated.`

Comment (adrianchiris) [reply pkg/tc/actuator_file_writer.go:77]: os.Create truncates or creates, and this writes to the file. it will overwrite old content as expected.

**Review by rollandf — APPROVED** [review]

**Review by moshe010 — APPROVED** [review]

**Pull request merged**